files/image_config/monit/conf.d/sonic-host

```diff
@@ -6,17 +6,17 @@
 ###############################################################################
 
 check filesystem root-overlay with path /
-    if space usage > 90% for 5 times within 10 cycles then alert
+    if space usage > 90% for 5 times within 20 cycles then alert
```

Comment:
You doubled the number of cycles which makes the interval consistent with the previous interval, but you didn't doble the number of "hits" required to alert. You also need to double those to maintain similar behavior (e.g., `if space usage > 90% for 5 times within 10 cycles then alert` with 120 second cycles ~= `if space usage > 90% for 10 times within 20 cycles then alert` with 60 second cycles).